openmetadata-ui/src/main/resources/ui/src/pages/APICollectionPage/APICollectionPage.tsx

```diff
@@ -454,13 +455,13 @@ const APICollectionPage: FunctionComponent = () => {
       if (cursorType) {
         getAPICollectionEndpoints({
           paging: {
-            [cursorType]: apiEndpoints.paging[cursorType],
+            [cursorType]: paging[cursorType],
           },
         });
       }
       setCurrentEndpointsPage(currentPage);
     },
-    [apiEndpoints, getAPICollectionEndpoints]
+    [paging, getAPICollectionEndpoints, handlePageChange]
```

Comment:
We are not using handlePageChange in the function so i dont think we need as dependancy

Reply:
Removed handlePageChange dependency.